tarofreighter/chain_porter.go

```diff
@@ -287,12 +287,12 @@ func (p *ChainPorter) waitForTransferTxConf(pkg *sendPackage) {
 	return
 }
 
-// storeAndTransferProofs writes the sender and receiver proof files to the proof
-// archive and then attempts to deliver a proof to the receiver.
-func (p *ChainPorter) storeAndTransferProofs(pkg *sendPackage) {
+// storeProofs writes the updated sender and receiver proof files to the proof
+// archive.
+func (p *ChainPorter) storeProofs(pkg *sendPackage) {
```

Comment:
Posting here just to have a place: I think one thing we might want to do before such refactoring is: get a better set of unit tests in place, to ensure that we're not inadvertently changing behavior we'd like to retain. The first half of #217 should help us achieve such a goal.

Reply:
I agree. This PR does now improve the testing situation a little. The basic send integration test should now include proof transfer via the local aperture instance.